topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+>
+> Another way to download datasets is to use CAMI Client. [Here](https://www.microbiome-cosi.org/cami/resources/cami-client) you can find instructions on how to use it.
+>
+> > ### {% icon hands_on %} Hands-on: Get dataset with DOI
+> > 
+> > 1. Install Java 8 on your computer.
+> > 2. Create an account on[ CAMI portal](https://data.cami-challenge.org/)
+> > 3. Download the [camiClient.jar](https://data.cami-challenge.org/camiClient.jar)
+> > 4. Get text-file with links for dataset from [CAMI portal](https://data.cami-challenge.org/). You can download this file in the section “2nd CAMI Challenge Marine Dataset”
+> > 5. You can download data with CAMI client using the following command line in the terminal:
+> > 
+> >     ```
+> > java -jar camiClient.jar -d <linkfileLocation> <targetDirectory> -p PATTERN
+> > ```
+> {: .hands_on}
+> 
+{: .details}
+
+
+#### Upload data into Galaxy
+
+There are different ways to upload data into the Galaxy. In case of a not very large dataset it’s more convenient to upload data directly from your computer to Galaxy. In this tutorial we use a large dataset, thus, we use FTP server for this purpose.
+
+> ### {% icon hands_on %} Hands-on: Upload data into Galaxy with FTP
+>
+> 1. Create new history
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Upload datasets into FTP Galaxy server
+>
+>    {% snippet faqs/galaxy/datasets_upload_ftp.md %}
+>
+> 3. Press “upload data” -> “choose remote files” -> “FTP directory” and upload your data to your history from FTP server
+{: .hands_on}
+
+
+
+#### Preprocessing data
+
+We got the marine data in our history, with one file per sample (10 samples for long reads and 10 samples for short reads). We need to prepare our data.
+
+
+
+1. We should organise our data into collections. For deeper learning about dataset collections follow {% cite noauthor_galaxy_nodate-1 %} tutorial.
+2. These files consist of interleaved paired-end reads, i.e. forward and reverse reads are all together in one single file and not 2 as often. This can be a problem for some assemblers as not every assembling tool can take interleaved paired-end reads as input.For co-assembly of all samples, some assemblers expect one file with all samples  combined together.
+
+> ### {% icon hands_on %} Hands-on: Preprocessing data
+>
+> 1. Take files for short reads for all samples
+> 2. Create a collection with all samples- We now have a collection with all files. Let's now combine all sample files into one file
+> 3. Combine all samples from collection into one file -  follow the section “[Collapse data into a single dataset](https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/collections/tutorial.html#collapse-data-into-a-single-dataset)” in {% cite noauthor_galaxy_nodate-1 %}
+> 
+>      We can now split forward and reverse reads into 2 files
+> 
+> 4. **FASTQ splitter** {% icon tool %} with following parameters:
```

Comment:
If you could use the new tool linking syntax it would be really nice for users! Then they could use "Tutorial Mode"

We have a [guide on how to do it](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links), but in short just switch to `{% tool [Your Tool](toolshed.g2.bx.psu.edu/repos/..../your-tool/1.0.0+galaxy0) %}`

Reply:
ok, good idea